mmv1/third_party/terraform/transport/config.go.tmpl

```diff
@@ -429,9 +482,34 @@ func (c *Config) LoadAndValidate(ctx context.Context) error {
 
 	c.Context = ctx
 
-	tokenSource, err := c.getTokenSource(c.Scopes, false)
-	if err != nil {
-		return err
+	var tokenSource oauth2.TokenSource
```

Comment:
This is ugly and repetitive at the moment - needs refactor when PoC is taken forward